src/strategies/python.ts

```diff
@@ -117,6 +117,22 @@ export class Python extends BaseStrategy {
           version,
         }),
       });
+      // In case the project name is splitted by -, replace - with _
+      const projectDirectoryName = projectName.replace('-', '_');
```

Comment:
I wonder if something weird will happen if `projectDirectoryName` and `projectName` are the same. E.g. will the later API code be weird if there are 2 updates to the same file.

Another approach we could do is to use a `Set`, add both `projectName` and `projectDirectoryName`, then iterate over the members and add the 2 paths as a way to de-dup.